Parent/Parent/Details/AccountNotificationViewController.swift

```diff
@@ -89,6 +89,14 @@ class AccountNotificationViewController: UITableViewController {
         tableView.reloadData()
     }
 
+    override func viewWillAppear(_ animated: Bool) {
+        super.viewWillAppear(animated)
+        if let studentID = currentStudentID {
+            let scheme = ColorCoordinator.colorSchemeForStudentID(studentID)
+            navigationController?.navigationBar.useContextColor(scheme.mainColor)
```

Comment:
Would it be helpful to put these 2 lines in an extension / protocol?  Cool too if it could take a optional studentID and do the checking on the extension / protocol.

Reply:
We could do that. It will only remove 1 line of code at each of these though, so 🤷‍♂.